enhancements/monitoring/user-workload-monitoring.md

```diff
@@ -360,8 +356,6 @@ embedding `get/create/edit/delete` permissions.
 User workload monitoring is not enabled by default. It must be explicitly enabled by the user.
 User workload monitoring is enabled just by creating an empty `workload-monitoring-config` configmap in the `openshift-monitoring` namespace.
 
-The user workload monitoring configuration allows to set all existing configuration options as for the in-cluster stack with the addition to set the replica count of the user workload monitoring Prometheus instances.
```

Comment:
> The user workload monitoring configuration allows to set all existing configuration options

As we are changing the configuration of user monitoring stack this might not hold true anymore.